npm_and_yarn/spec/dependabot/npm_and_yarn/file_updater_spec.rb

```diff
@@ -3978,7 +3978,7 @@
         it "updates the manifest and lockfile" do
           expect(updated_files.map(&:name)).to match_array(%w(package.json pnpm-lock.yaml))
 
-          expect(updated_pnpm_lock.content).to include("/node-adodb@5.0.2:")
+          expect(updated_pnpm_lock.content).to include("/node-adodb@5.0.3:")
```

Comment:
This is the result of the following change : 


> The default value of the resolution-mode setting is changed to highest. This setting was changed to lowest-direct in v8.0.0 and some users were https://github.com/pnpm/pnpm/issues/6463. A [twitter poll](https://twitter.com/pnpmjs/status/1693707270897517022) concluded that most of the users want the old behaviour (resolution-mode set to highest by default). This is a semi-breaking change but should not affect users that commit their lockfile https://github.com/pnpm/pnpm/issues/6463.

Reply:
@deivid-rodriguez Your input would be appreciated here 🙇

Reply:
@deivid-rodriguez just headed out on holiday and will be offline for a few weeks (I think he might be 🚴 ?)

But I agree it'd be good to get his input on this, so if no one else on the team picks this up and you don't hear from him  by Sept 20th, feel free to ping him again.

Reply:
That sounds fine @yeikel, but we may want to make the test resilient to new releases of this dependency. Maybe by using an exact dependency? This test claims to be dealing with incorrect platform dependencies, so I guess such as change would not affect that?

Reply:
Given the feedback that they received, I do not expect this to change again anytime soon

Reply:
@deivid-rodriguez What do you suggest we do now?

Reply:
Do you mean you don't expect upstream PNPM to change default resolution mode again soon, or you don't expect `node-adodb` to release 5.0.4 anytime soon?

My suggestion was aimed at making this spec resilient to a potential release of node-adodb 5.0.4.